azerothcore/azerothcore-wotlk
# fix(DB/creature): Ulduar 25 extra trash packs

## CHANGES PROPOSED:
In the 25-man version of Ulduar, there should be 2 packs of Iron Dwarf trash at the start of The Antechamber and at the base of the stairs leading to Kologarn.


## TESTS PERFORMED:
<details><summary>Before</summary>

![image](https://user-images.githubusercontent.com/61268368/99883776-edba5a80-2bff-11eb-9a5d-e328670dc083.png)

![image](https://user-images.githubusercontent.com/61268368/99883781-f743c280-2bff-11eb-9c2d-7d0cecb2c7b5.png)

</details>
<details><summary>After</summary>

![image](https://user-images.githubusercontent.com/61268368/99883802-0cb8ec80-2c00-11eb-8fe2-827eb75b7ea3.png)

![image](https://user-images.githubusercontent.com/61268368/99883807-12163700-2c00-11eb-80db-1e465a334bc0.png)

</details>

## HOW TO TEST THE CHANGES:
Set raid difficulty 10-player
.go gob 34935
Confirm 1 pack of Iron Dwarf trash near teleporter and at the stairs leading to Kologarn.

Set raid difficulty 25-player
.go gob 34935
Confirm 2 packs of Iron Dwarf trash near teleporter and at the stairs leading to Kologarn.


## Target branch(es):


## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
